R/mod_safetyGraphicsServer.R

```diff
@@ -40,25 +42,31 @@ safetyGraphicsServer <- function(
         filterDomain=filterDomain, 
         current_mapping=current_mapping
     )
-    
-    callModule(homeTab, "home", config)
 
     #Initialize Chart UI - Adds subtabs to chart menu - this initializes initializes chart UIs
     charts %>% purrr::map(~chartsNav(.x,session$ns))
 
     #Initialize Chart Servers
-    validDomains <- tolower(names(mapping))
-    charts %>% purrr::map(
-        ~callModule(
-            module=chartsTab,
-            id=.x$name,
-            chart=.x,
-            data=filtered_data,
-            mapping=current_mapping    
-        )
-    )
+    module_outputs <- reactiveValues()
```

Comment:
delete this?

Reply:
`reactiveValues` is actually what's working. It allows passing around a single, updateable object of reactive modules.